component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/internal/X509CertificateServiceComponent.java

```diff
@@ -99,4 +108,31 @@ protected void unsetHttpService(HttpService httpService) {
             log.debug("HTTP Service is unset in the X509 Certificate Servlet");
         }
     }
+
+    protected void setAccountLockService(AccountLockService accountLockService) {
+
+        X509CertificateDataHolder.getInstance().setAccountLockService(accountLockService);
+    }
+
+    protected void unsetAccountLockService(AccountLockService accountLockService) {
+
+        X509CertificateDataHolder.getInstance().setAccountLockService(null);
+    }
+
+    protected void setRealmService(RealmService realmService) {
+
+        if (log.isDebugEnabled()) {
+            log.debug("RealmService is set the X509 authenticator bundle.");
+        }
+
```

Comment:
Is this newline needed?

Reply:
Removed with ed2968b2c84d173790ff71b500a0268434981d01